src/test/java/dk/cngroup/trainings/spring/springassignment/controller/AnimalControllerTest.java

```diff
@@ -37,6 +37,8 @@
 @RunWith(SpringRunner.class)
 public class AnimalControllerTest {
```

Comment:
I know that this is just early stage, but even for the MVC controllers, there is a nice way in Spring how to test them. So don't forget to add tests.

You can still use the MockMvc, but instead of checking `jsonPath`. You can check what view is used, what is in the model, what is not in the model, etc. Some examples: https://www.petrikainulainen.net/programming/spring-framework/unit-testing-of-spring-mvc-controllers-normal-controllers/